types/google-spreadsheet/index.d.ts

```diff
@@ -768,10 +768,11 @@ export class GoogleSpreadsheetWorksheet implements WorksheetBasicProperties {
 
     /**
      * @description
-     * loads the header row (first row) of the sheet
+     * loads the header row (first by default) of the sheet
      * - usually do not need to call this directly
+     * - Attention: rows indexed from 1
      */
-    loadHeaderRow(): Promise<void>;
+    loadHeaderRow(headerRowIndex?: number): Promise<void>;
```

Comment:
I would add a `@param` JSDoc, and keep the doc comments the same as [on the website](https://theoephraim.github.io/node-google-spreadsheet/#/classes/google-spreadsheet-worksheet?id=fn-loadheaderrow)